src/App/WorkerStats/WorkerStats.js

```diff
@@ -1,29 +1,35 @@
 import React from 'react';
-import { workerStats } from '../../../test/workerStats'
+import styles from './WorkerStats.css';
+import MinerInfo from '../MinerInfo/MinerInfo';
 
 export const WorkerStats = () => {
-    const { mature, immature, totalHash, totalShares, paid } = workerStats;
-    const miner = workerStats.miner.split('=');
-    
+    const minerData= {
+        'Total Miners': 1234,
+        'Hashrate': 2345,
+        'Accepted': 3456,
+        'Rejected': 4567
+    };
     return (
         <div>
-            <div>
-                Miner: {miner[1]}
+            <div className={styles.minerStatsTop}>
+            {/* these will probably be components with props passed in or something */}
+                <div className={styles.minerInfo}>
+                    <MinerInfo title='Miner Information' data={minerData} />
```

Comment:
To create additional comonent independence, we should avoid passing data from parent to child unless its static (e.g. title='Miner Information') or its the result of mapping over an array, and creating components that way. 

Also, any objection to moving the wrapper div to within the Component? Ideally I think this could read a bit closer to:
 <MinerInfo title='Miner Information' />
<MinerInfo title='Balance' />
<MinerInfo title='Pool Information/>

However, now that I think about it more, I think we talked about making these 3 separate Components, each named the title.